nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Update/UpdateWorkerTests.Sdk.cs

```diff
@@ -16,6 +16,7 @@ public class Sdk : UpdateWorkerTestBase
         [InlineData("net472")]
         [InlineData("net7.0")]
         [InlineData("net8.0")]
+        [InlineData("net9.0")]
```

Comment:
If this test were skipped, the need to update the updated to `net9.0` itself can be reverted. Otherwise, the tests need reworking in some way to discover .NET SDK versions without relying on the exact version of .NET the tests are executing with.